### DISCUSSION TITLE
Add a 'Random Model' button

### DISCUSSION THREAD

**Discussion by QwertyITA** [civitai/civitai#219] [Ideas]
Add a 'Random Model' button

**Comment by JustMaier** [comment]
Like the good old "I'm Feeling Lucky" button on Google?

**Reply by EsterLoken** [reply]
![image](https://user-images.githubusercontent.com/33354862/219995466-32a1b93c-4fb4-42d9-965e-75560da7af97.png)

yeah, I'd love to have it as an option in the top left dropdown. With RANDOM sorting active, every page refresh should return a new, completely random list of models/embeddings/loras etc. depending on the filter settings in the top right.

I feel like the options most downloaded/liked/discussed/rated have become so stable now, they're basically the same page, at least at the top.

With RANDOM sorting it would be a bit like panning for gold, one might find a cool model that would otherwise be impossible to come across after it dropped off of NEWESTs fleeting radar.

**Reply by WASasquatch** [reply]
> Like the good old "I'm Feeling Lucky" button on Google?

Ahh yes, the good ol' "I'm ready to be traumatized" button.
